.github/workflows/contributors.yml

```diff
@@ -0,0 +1,16 @@
+name: Contributors
+on:
+  schedule:
+    - cron: '0 1 * * *' # At 01:00 on every day.
+  push:
+    branches:
+      - rolling
+jobs:
+  contributors:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wow-actions/contributors-list@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          round: true
+          svgPath: ../../images/contributors_list.svg
```

Comment:
I can't find where we use this svg. Do we need to use it in the Readme.md?

Reply:
Yes :slightly_smiling_face: , we need to use the generated `svg file` in the `Readme.md` if you want to like to know more about the GitHub action you can check this [out](https://github.com/marketplace/actions/contributors-list) :slightly_smiling_face: .

Reply:
I see ~